backend/.env.example

```diff
@@ -4,3 +4,8 @@ MONGODB_CONNECTION_STRING=mongodb+srv://username:password@cluster0.example.mongo
 AUTH0_AUDIENCE=your-auth0-audience
 # Issuer Base URL
 AUTH0_ISSUER_BASE_URL=https://your-auth0-domain/
+
+# Cloudinary
+CLOUDINARY_CLOUD_NAME=your-cloud-name
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/26#issuecomment-2382238808>#d993e8</a><span> - 09/30/2024, 06:50 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The .env.example file contains placeholder values that might be mistaken for real credentials, potentially leading to security risks if developers accidentally use these values in production.
</div>
<div id="fix">
<b>Fix:</b> Replace the placeholder values with clear instructions or use a format that cannot be mistaken for real credentials.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -7,5 +7,5 @@ AUTH0_ISSUER_BASE_URL=https://your-auth0-domain/

  # Cloudinary
 -CLOUDINARY_CLOUD_NAME=your-cloud-name
 -CLOUDINARY_API_KEY=your-api-key
 -CLOUDINARY_API_SECRET=your-api-secret
 +CLOUDINARY_CLOUD_NAME=<your_cloud_name>
 +CLOUDINARY_API_KEY=<your_api_key>
 +CLOUDINARY_API_SECRET=<your_api_secret>
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged